### DISCUSSION TITLE
連続する同一記事へのいいねやストックの通知を適当にまとめてほしい

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#569] [要望・アイデア・不具合]
同一記事のいいねやストックに関する通知が連続する場合、それらだけで 🔔 の通知領域を占領してしまい、他の記事に関するリアクションを見逃す可能性があります。

連続した同じような通知は適当にまとめられていると見やすくなってありがたいです。

例えば Twitter X では、同一のツイートに関する通知は適当にまとめられているため、連続して同一種の通知が大量に来ても通知欄を見るとそれなりに対応することが出来ます。

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
リクエストありがとうございます！
今後の改善の参考にさせていただきます🙏
